src/engraving/rendering/score/tlayout.cpp

```diff
@@ -3508,6 +3508,11 @@ void TLayout::layoutKeySig(const KeySig* item, KeySig::LayoutData* ldata, const
 //        return;
 //    }
 
+    // We do not show keys if the key is for instrument change and we are in concert pitch mode
+    if (item->forInstrumentChange() and item->concertPitch()) {
```

Comment:
Really `and` rather than `&&`?

Reply:
FWIW, these `and` `or` `not` operators really exist: https://en.cppreference.com/w/cpp/language/operator_alternative
They are funny, but let's stick to the normal ones... :)